### PR TITLE
open-zwave 1.4

### PR DESCRIPTION
**Pull request by oandrew**
Update open-zwave to 1.4 + HEAD

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by ilovezfs** [comment]
Have the issues from https://github.com/Homebrew/homebrew-core/pull/3268 been resolved?

File: Formula/open-zwave.rb [hunk 1]
```diff
@@ -1,132 +1,32 @@
 class OpenZwave < Formula
   desc "Library for selected Z-Wave PC controllers"
-  homepage "http://www.openzwave.com"
-  url "http://old.openzwave.com/downloads/openzwave-1.2.919.tar.gz"
-  sha256 "473229f3dd3d6b260e6584b17e5c5f2e09e61805f89763f486a9f7aa2b4181ba"
+  homepage "http://www.openzwave.net/"
+  url "https://github.com/OpenZWave/open-zwave/archive/v1.4.tar.gz"
```
Comment (ilovezfs) [inline Formula/open-zwave.rb]: why the switch from the canonical URL?

File: Formula/open-zwave.rb [hunk 2]
```diff
@@ -1,132 +1,32 @@
...
+  sha256 "efb54adf0cda12f76fcb478f7cf3b7d108b5daf38d60b4410cca7600aa072e83"
+  head "https://github.com/OpenZWave/open-zwave.git"
```
Comment (ilovezfs) [inline Formula/open-zwave.rb]: are you planning to use `--HEAD` personally?

File: Formula/open-zwave.rb [hunk 3]
```diff
@@ -1,132 +1,32 @@
...
 
-  bottle do
```
Comment (ilovezfs) [inline Formula/open-zwave.rb]: please don't modify or remove the bottle block manually

File: Formula/open-zwave.rb [hunk 4]
```diff
@@ -1,132 +1,32 @@
...
-  # https://code.google.com/p/open-zwave/source/detail?r=954
-  patch :DATA
+  depends_on "pkg-config" => :build
+  depends_on "doxygen" => :build
 
   def install
     ENV["BUILD"] = "release"
     ENV["PREFIX"] = prefix
-
+    ENV["pkgconfigdir"] = "#{lib}/pkgconfig"
```
Comment (ilovezfs) [inline Formula/open-zwave.rb]: This shouldn't be necessary

**Review by ilovezfs — CHANGES_REQUESTED** [review]

File: Formula/open-zwave.rb (hunk 1 above)
Comment (oandrew) [reply Formula/open-zwave.rb]: github openzwave repo has a tag only for 1.4 (without minor versions) so it was even hard to understand what is  `1.4.164` version  from old.openzwave.com.
Also the `old` prefix in `old.openzwave.com`  made me feel like it was some legacy or unused site.

File: Formula/open-zwave.rb (hunk 2 above)
Comment (oandrew) [reply Formula/open-zwave.rb]: Yep. 
openzwave includes xml definitions for devices and those are updated quite often.
https://github.com/OpenZWave/open-zwave/tree/master/config

File: Formula/open-zwave.rb (hunk 3 above)
Comment (oandrew) [reply Formula/open-zwave.rb]: ok will put it back

File: Formula/open-zwave.rb (hunk 4 above)
Comment (oandrew) [reply Formula/open-zwave.rb]: I wish it wasn't but it is pretty messy.
https://github.com/OpenZWave/open-zwave/blob/master/cpp/build/support.mk#L103-L108

The only other way is to add `mkdir "#{lib}/pkgconfig"` before the build which looks worse to me

File: Formula/open-zwave.rb (hunk 2 above)
Comment (ilovezfs) [reply Formula/open-zwave.rb]: OK.

File: Formula/open-zwave.rb (hunk 1 above)
Comment (ilovezfs) [reply Formula/open-zwave.rb]: Please file an issue upstream and ask them what the correct, current stable URL is.

File: Formula/open-zwave.rb (hunk 4 above)
Comment (ilovezfs) [reply Formula/open-zwave.rb]: This needs to be reported and fixed by upstream.

**Pull request closed without merging**